packages/config/config/devices/0x031e/templates/inovelli_templates.json

```
@@ -0,0 +1,395 @@
+{
+	"hue_color_wheel_white": {
+		"minValue": 0,
+		"maxValue": 255,
+		"unsigned": true,
+		"description": "Calculated by hue color wheel (hue/360*254); 255 = White",
```

For consistency with other configs:
```suggestion
			"description": "Uses a scaled hue value (realHue / 360 * 254).",
```
I don't think we need to explicitly mention white in the description when there are options.